tutorials/vr/developing_for_oculus_quest.rst

```diff
@@ -0,0 +1,124 @@
+.. _doc_developing_for_oculus_quest:
+
+Introduction
+------------
+
+This tutorial goes over how to get started developing for the
+Oculus Quest with an official plugin.
+
+Before getting started there are two things you need to do.
+
+First you need to go through the steps on the :ref:`doc_exporting_for_android`
+page. This leads you through installing the toolset that Godot
+needs to export to Android devices.
+
+Next you need the Quest Plugin. You can download it from the Asset
+Library or manually download it from `here <https://github.com/GodotVR/godot-oculus-mobile-asset>`__.
+
+Setting Up Godot
+----------------
+
+To get started Open up Godot and create a new project
+
+.. image:: img/quest_new_project.png
+
+Make sure to chose the GLES2 renderer. Due to optimisations in the
+Quest's GPU this backend is far better suited for the Quest.
+
+Copy the addons folder from the Oculus Mobile asset into your Godot
+project. Your project tree should looks something like this:
+
+.. image:: img/quest_project_tree.png
+
+Now you can start building the main scene, stick to the bare minimum.
+
+- Add an ARVROrigin node first. 
+- Then Add three child nodes to the origin node, one ARVRCamera and two ARVRControllers.
+- Assign controller id 1 to the first ARVRController and rename that to LeftHand.
+- Assign controller id 2 to the second ARVRController and rename that to RightHand.
+- Finally add a MeshInstance as a child node to our first ARVRController and create a box shape, resize the box so each side is set to 0.1. Now duplicate the MeshInstance and move it to the second ARVRController node. These will stand in for our controllers, we're keeping things really simple.
+
+.. image:: img/quest_scene_tree.png
+
+Now add a script to the main node and add the following code:
+
+.. tabs::
+ .. code-tab:: gdscript GDScript
+
+    extends Spatial
+
+    var perform_runtime_config = false
```

Comment:
When you see `perform_runtime_config` you don't think this is boolean it sounds more like a function name to do action 

Maybe you could rename to `can_perform_runtime_config` to imply in name that this can either be true or false?